readme.md

```diff
@@ -1,96 +1,64 @@
-# Bitbucket Server and Jenkins integration
+# Bitbucket Server integration
 [![Build Status](https://ci.jenkins.io/job/Plugins/job/atlassian-bitbucket/job/master/badge/icon)](https://ci.jenkins.io/job/Plugins/job/atlassian-bitbucket-integration/job/master/)
```

Comment:
@khughes-atlassian @dkjellin this build status doesn't work, but i couldn't figure out how to fix it. Please let me know what I need to do.